lib/build.js

```diff
@@ -311,17 +321,32 @@ module.exports = {
         this.child.on('close', function(exitCode) {
           this.buildView.buildFinished(0 === exitCode);
           if (0 === exitCode) {
+            // if the command has a linter, clear lints for successful builds.
+            if (this.cmd.linter !== undefined) {
```

Comment:
It seems very wrong to have one linter per command. I thinks it feels more logical if there is one linter which is used for whatever command is run, and cleared between runs.

This would also help remove the Rust specific code in `linthelper.js` and the `Cargo` build tool